source/normalize-arguments.js

```diff
@@ -8,6 +8,7 @@ const urlToOptions = require('./url-to-options');
 const isFormData = require('./is-form-data');
 
 const RETRY_AFTER_STATUS_CODES = new Set([413, 429, 503]);
+const knownHookEvents = ['beforeRequest'];
```

Comment:
This should be named `KNOWN_HOOK_EVENTS` to keep the consistency of the code style :)

Reply:
I saw `RETRY_AFTER_STATUS_CODES` was chosen recently in #508, but I'm wondering if y'all would like to reconsider that choice and make it `retryAfterStatusCodes` as was the alternative suggestion in that discussion.

This is the only place in this package where this naming style is used. I know that uppercase constants is conventional in some languages, but not JS. Further, having two styles for variable names requires a decision to be made about when to use the uppercase style. Is it for all vars declared as `const`? It is for `const` outside of a function?

Reply:
@jstewmon I agree. I feel like I never know when to put it in uppercase. Better to just always use normal camelcase, indeed.